scripts/crab/readaMCatNloEntries.py

```diff
@@ -76,7 +78,9 @@ def readEntries(worker, xmlfiles, fast=False):
         pool.join()
         entries_per_rootfile = [r for r in result.get()]
         print "number of events in",xml,sum(entries_per_rootfile)
+	result_list.append(sum(entries_per_rootfile))
```

Comment:
Can you fix the spacing here please? Looks like tab & spaces got mixed up - on github the indentation looks weird